javascript/javascript2/week1/shortest-word-in-an-array.js

```diff
@@ -0,0 +1,16 @@
+const danishWords = ['bil', 'plante', 'kaffe', 'bog', 'ø', 'planetarium'];
+
+function shortestWord(array) {
+  let shortest = '';
+
+  for (let element of array) {
```

Comment:
Try to use meaningfull names in your code. For example
```suggestion
  for (let word of words) {
```